cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityOperator.java

```diff
@@ -155,6 +157,13 @@ public static EntityOperator fromCrd(Kafka kafkaAssembly) {
                     result.templateSecurityContext = pod.getSecurityContext();
                 }
             }
+
+            KafkaClusterSpec kafkaClusterSpec = kafkaAssembly.getSpec().getKafka();
+            String tlsSidecarImage = versions.kafkaImage(kafkaClusterSpec.getImage(), kafkaClusterSpec.getVersion());
```

Comment:
Does this mean we will have to update the EO pod everytime Kafka version changes? Wouldn't it be easier to just use the image for the default version?